.gitignore

```diff
@@ -1,2 +1,3 @@
 node_modules
 .env
+myInfo.env
```

Comment:
[IMO] Instead of using myInfo.env, create .env file to your local which is never committed.

Reply:
[IMO] also you don't use any key in this branch, so you don't need env file now.

Reply:
Yeah I made the mistake of committing the .env file, but thanks for the critique. For the image, ill convert it to the wanted format.

Reply:
I mean you can delete myInfo.env in gitignore and this project. because you never use it. You use .env instead. Your file will be like this. 
<img width="232" alt="Screen Shot 2022-11-08 at 15 02 41" src="https://user-images.githubusercontent.com/75482698/200694770-84a7d2a2-6021-4093-98de-565050420c32.png">